hermes-consumers/build.gradle

```diff
@@ -10,12 +10,14 @@ dependencies {
     compile group: 'org.eclipse.jetty', name: 'jetty-client', version: '9.3.6.v20151106'
 
     compile group: 'javax.jms', name: 'javax.jms-api', version: '2.0'
-    compile group: 'com.jayway.jsonpath', name: 'json-path', version: '0.9.1'
+    compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.2.0'
```

Comment:
you have already defined this dependency in hermes-common, therefore you don't need to add it here again